examples/kotlin/src/main/kotlin/com/xebia/functional/xef/conversation/conversations/Animal.kt

```diff
@@ -29,7 +30,7 @@ suspend fun main() {
     messagePolicy { addMessagesFromConversation = MessagesFromHistory.NONE }
   }
 
-  OpenAI.conversation {
+  OpenAI.conversation(metric = LogsMetric()) {
```

Comment:
Do we want to use `LogMetric()` here? Or should we change it to `Metric.EMPTY`?

Reply:
In this case, as it's an example, I think we can use this to see the logs.